src/libAtomVM/term.h

```diff
@@ -258,6 +277,36 @@ static inline int term_is_binary(term t)
     return 0;
 }
 
+/**
+ * @brief Checks if a term is a binary
+ *
+ * @details Returns 1 if a term is a binary stored on the heap, otherwise 0.
+ * @param t the term that will be checked.
+ * @return 1 if check succedes, 0 otherwise.
+ */
+static inline int term_is_refc_binary(term t)
+{
+    /* boxed: 10 */
+    if ((t & 0x3) == 0x2) {
```

Comment:
we can replace this with `if (term_is_boxed(t)) {`

Reply:
+1

Reply:
+1

Reply:
Done.